source/cloud/aws/ec2-multi.md

```diff
@@ -20,7 +20,7 @@ Install the AWS CLI tools following the [official instructions](https://docs.aws
 
 Also install `dask-cloudprovider` and ensure you select the `aws` optional extras.
 
-```console
+```bash
 $ pip install "dask-cloudprovider[aws]"
```

Comment:
When it's `bash` you omit the `$`.

Reply:
Is there a particular reason for that? Because we have examples in the docs of is using `$` with bash 

https://github.com/rapidsai/deployment/blob/03c86b0b846e5a0e7e36fa4e0bf624b7f0c0139f/source/tools/kubernetes/dask-helm-chart.md?plain=1#L117-L123

For reference, when we copy this with the button it only copies the command.